### PR TITLE
Check server status after window.open to prevent pop-up blocking in browser

### PR DESCRIPTION
**Pull request by dmohs**
Fixes bug in GAWB 808.
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] **Submitter**: Tell tech lead that the PR exists if s/he wants to look at it
- [x] **Submitter**: Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x] **Tech lead**: sign off
- [x] **LR**: sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't 
  working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev server still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
This works for me as a quick fix, but I don't like how the popup window shows the footer and the main application skin while checking server status. Can we have a follow-on story to deal with that?

**Comment by dmohs** [comment]
@davidangb [Agreed](https://broadinstitute.atlassian.net/browse/GAWB-927).

File: src/cljs/org/broadinstitute/firecloud_ui/main.cljs [hunk 1]
```diff
@@ -406,79 +406,84 @@
...
-                                       (or success? (= status-code 404))
-                                       (react/call :handle-successful-auth this token)
-                                       :else
-                                       (swap! state assoc :user-status :error))))
-               (swap! state assoc :user-status :not-logged-in)))))))
+     (if (= (.. js/window -location -hash) sign-in/flow-start-location-hash)
+       (utils/ajax-orch
+        "/me"
+        {:service-prefix ""
+         :on-done (fn [{:keys [success? status-code]}]
```
Comment (skwalker) [inline src/cljs/org/broadinstitute/firecloud_ui/main.cljs]: need comment as to why not checking success flag

**Comment by skwalker** [comment]
👍

**Pull request merged**